cookbook/starter-apps/Educhain_pedagogy/Backend/app/models/pedagogy_models.py

```
@@ -0,0 +1,22 @@
+from pydantic import BaseModel
+from typing import Dict, Any, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)